src/config/reader.rs

```diff
@@ -117,7 +107,7 @@ impl ConfigReader {
                     if !config.links.is_empty() {
                         config_module = config_module.merge_right(
                             &self
-                                .ext_links(ConfigModule::from(config), Some(source.path))
+                                .ext_links(ConfigModule::from(config), parent_dir)
```

Comment:
is this part right? it was path to source of the link itself with probably its parent_dir, but now we reuse the same parent_dir. It's probably related to case for links inside already linked config